src/commands/manifest.ts

```diff
@@ -47,25 +53,33 @@ export default class Manifest extends Command {
       const tmpDir = os.tmpdir()
       const promises = Object.entries(packageJson.oclif.jitPlugins).map(async ([jitPlugin, version]) => {
         const pluginDir = jitPlugin.replace('/', '-').replace('@', '')
-        const repo = this.executeCommand(`npm view ${jitPlugin}@latest repository --json`)
-        const stdout = JSON.parse(repo.stdout)
-
-        const repoUrl = stdout.url.replace(`${stdout.type}+`, '')
 
         const fullPath = path.join(tmpDir, pluginDir)
+
         if (await fileExists(fullPath)) await fs.remove(fullPath)
 
-        const versions = JSON.parse(this.executeCommand(`npm view ${jitPlugin}@latest versions --json`).stdout)
-        const maxSatisfying = semver.maxSatisfying(versions, version)
+        await fs.mkdir(fullPath, {recursive: true})
 
-        this.cloneRepo(repoUrl, fullPath, maxSatisfying)
+        const tarballUrl = this.getTarballUrl(jitPlugin, version)
+        const tarball = path.join(fullPath, path.basename(tarballUrl))
+        await pipeline(
+          got.stream(tarballUrl),
```

Comment:
if anyone else in the oclifiverse ever starts doing JIT stuff, and is building behind a proxy or via CI, they'll have to ask us to enable proxy support for this.

Reply:
Possibly. Although this exact same technique is used for downloading and extracting the node executable, which hasn't been an issue for anyone yet